sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -347,14 +347,14 @@ def finish_bundle(self):
 
     pipeline = TestPipeline()
     result = (pipeline
-              | 'Start' >> beam.Create([x for x in range(3)])
+              | 'Start' >> beam.Create([1])
               | beam.ParDo(MyDoFn())
               | WindowInto(windowfn)
               | 'create tuple' >> beam.Map(
                   lambda v, t=beam.DoFn.TimestampParam, w=beam.DoFn.WindowParam:
                   (v, t, w.start, w.end)))
-    expected_process = [('process'+ str(x), Timestamp(5), Timestamp(4),
-                         Timestamp(6)) for x in range(3)]
+    expected_process = [('process'+ str(1), Timestamp(5), Timestamp(4),
```

Comment:
```suggestion
    expected_process = [('process1', Timestamp(5), Timestamp(4),
```

Reply:
@liumomo315 
Shall I add this suggestion to the change?

Reply:
it's not important I think. we can merge without.

Reply:
Sorry I missed this comment earlier. Committed the suggestion. Thanks!